docs/docs/configuration.md

```diff
@@ -131,29 +122,27 @@ cardano-cli address build --verification-key-file cardano.vk --mainnet
 
 <!-- TODO: this part below feels ab it odd here, rather move to API or how-to? -->
 
-While the `hydra-node` needs to pay fees for protocol transactions, any wallet can be used to commit **funds** into an `initializing` Hydra head. The `hydra-node` provides an HTTP endpoint at `/commit`, which allows to specify either:
- - a `UTxO` set of outputs to commit (belonging to public keys) or
- - a _blueprint_ transaction together with the `UTxO` which resolves it.
-
-and eventually returns a commit transaction.
+While the `hydra-node` needs to pay fees for protocol transactions, any wallet can be used to commit **funds** into an `initializing` Hydra head. The `hydra-node` provides an HTTP endpoint at `/commit`, allowing you to specify either:
+ - A set of `UTXO` outputs to commit (belonging to public keys), or
+ - A _blueprint_ transaction along with the `UTXO` that resolves it.
```

Comment:
We do call unspent transaction outputs `UTxO` (small x) everywhere in the code and documentation otherwise. Even in this document there are multiple other uses with small x.

Please keep things consistent as much as possible, where local consistency is even more important than abiding to some style guide IMO